apps/ima/views/risk-of-harm.html

```diff
@@ -1,5 +1,8 @@
 {{<partials-page}}
   {{$page-content}}
+    {{#noMoreHarmClaims}}
+      <p class="govuk-body"><strong>You will not be able to add another country as 5 have already been added. To add another, please delete one first.</strong></p>
```

Comment:
maybe add  'To add another, please **go to the summary** and delete one first'

Reply:
Amended the wording as suggested.